### PR TITLE
Added Prometheus node exporter role

### PR DESCRIPTION
**Pull request by jpds**

### CODE REVIEW

File: ansible/roles/node-exporter/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Example Playbook
+----------------
+
+The following playbook configures Node Exporter.
+
+    ---
+    - hosts: nodeexporter
+      roles:
+        - role: nodeexporter
```
Comment (markgoddard) [inline ansible/roles/node-exporter/README.md]: node-exporter

File: ansible/roles/node-exporter/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# Whether Node Exporter is enabled.
+nodeexporter_enabled: true
+
+# Service deployment definition.
+nodeexporter_services:
+  nodeexporter:
+    container_name: nodeexporter
+    enabled: "{{ nodeexporter_enabled }}"
+    image: "{{ nodeexporter_image_full }}"
+    command: /bin/node_exporter --collector.procfs=/host/proc --collector.sysfs=/host/sys --collector.filesystem.ignored-mount-points "^/(sys|proc|dev|host|etc)($|/)"
```
Comment (markgoddard) [inline ansible/roles/node-exporter/defaults/main.yml:13]: I don't see this being used.

File: ansible/roles/node-exporter/tasks/reconfigure.yml [hunk 3]
```diff
@@ -0,0 +1 @@
+deploy.yml
```
Comment (markgoddard) [inline ansible/roles/node-exporter/tasks/reconfigure.yml:1]: Should this be a symlink? Alternatively, use `- include: deploy.yml`.

**Review by markgoddard — COMMENTED** [review]
Mostly looks fine.

File: ansible/roles/node-exporter/tasks/reconfigure.yml (hunk 3 above)
Comment (jpds) [reply ansible/roles/node-exporter/tasks/reconfigure.yml:1]: This one I copied from the opensm role.

Comment (markgoddard) [reply ansible/roles/node-exporter/tasks/reconfigure.yml:1]: Right, but it's a symlink in that role, and copying doesn't appear to have done the right thing (unless I'm missing something?).

**Review by markgoddard — APPROVED** [review]
Looks fine.

File: ansible/roles/node-exporter/defaults/main.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
+---
+# Roughly follows kolla-ansible's service deployment patterns.
+
+# Whether Node Exporter is enabled.
+nodeexporter_enabled: true
```
Comment (markgoddard) [inline ansible/roles/node-exporter/defaults/main.yml]: I realise I've already approved this, but I looked again before merging and noticed this default. I'm not sure we want to enable this by default - it seems specific to the Verne deployment. Can we set it to false, and add an override in kayobe-config?

Comment (jpds) [reply ansible/roles/node-exporter/defaults/main.yml]: Sounds good to me, done.

**Review by markgoddard — APPROVED** [review]
Thanks.

**Pull request merged**